spec/models/like_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  before(:all) do
+    @user = User.create(name: 'test', photo: 'https://unsplash.com/photos/', bio: 'test')
+    @post = Post.create(author_id: @user.id, title: 'test', text: 'test', user_id: @user.id)
+  end
+
+  it 'should create a like' do
+    like = Like.create(author_id: @user.id, post_id: @post.id, user_id: @user.id)
+    expect(like).to be_valid
+  end
+end
```

Comment:
- [ ] Kindly remove `user_id: @user.id` as previously suggested and fix the `update_likes_counter` in the Like model.

  ![image](https://user-images.githubusercontent.com/54848363/213817905-b8d07128-c7f7-4a32-acf7-a3f4cbf7de2d.png)